### ISSUE TITLE
Mac OSX scroll down doesn't work

### ISSUE THREAD

**Issue by rakanalh** [NHDaly/tmux-scroll-copy-mode#23]
The error it shows is:
```
select-pane -t= ; if -t = \"/Users/rakan/.tmux/plugins//tmux-better-mouse-mode/only_scroll_sometimes.sh 2\" \"send-keys -M\" \"\"
```

I am running MacOS Sierra, tmux 2.3 and latest tmux-better-mouse-mode.

Could you please look into this?

Thanks,
Rakan

**Comment by NHDaly** [comment]
Hmm, hi! Thanks for the issue report.

I'm sorry i don't have MacOS Sierra yet. But I just upgraded my tmux from `tmux 2.2` to `tmux 2.3_2` in order to test this.

And yeah, I see now that if I change my `@scroll-speed-num-lines-per-scroll` to `0.5`, which I assume is how you have it, that yes I see that same error.

I'll look into why now!

**Comment by NHDaly** [comment]
But as a short work around you can set the scroll speed to `1` instead, which will alleviate the issue for you (but your scrolling will be a bit faster).

**Comment by NHDaly** [comment]
I think it has something to do with parsing the `'`s and `"`s prevalent in that nasty ish line. I'm trying to work out the hairy details, but I'm still not sure what's going on.

**Comment by NHDaly** [comment]
I'm going to sleep, but i'll try to look at this tomorrow. please do let me know if you are able to isolate or even fix the problem!

**Comment by rakanalh** [comment]
Hi @NHDaly,

Yes after some trials on debugging this issue i found out that setting to a 1 value fixes the issue for now. But i tried executing each part like only_scroll_sometimes.sh and i don't get any errors.

Thank you for looking into this.

**Comment by NHDaly** [comment]
Okay! I've fixed your error in 4c686e5a1f381e072b102df63f3f8a17426fb107! :)

Please go ahead and update the plugin through `tpm` and if it works correctly for you with your original, desired value of `@scroll-speed-num-lines-per-scroll`, please close this bug! :) Thanks for reporting it!

**Comment by rakanalh** [comment]
Tested and it works perfectly.

Thanks alot!
